README.md

```diff
@@ -1,38 +1,58 @@
-![logo](tokbox-logo.png)
+# OpenTok Accelerator TextChat for JavaScript <br/>
+[![Build Status](https://travis-ci.org/opentok/accelerator-textchat-js.svg?branch=master)](https://travis-ci.org/opentok/accelerator-textchat-js) 
+[![npm](https://img.shields.io/npm/v/opentok-text-chat.svg)](https://www.npmjs.com/package/opentok-text-chat) 
+[![license Apache](https://img.shields.io/github/license/opentok/accelerator-textchat-js.svg)](./.github/LICENSE)
 
-[![Build Status](https://travis-ci.org/opentok/accelerator-textchat-js.svg?branch=master)](https://travis-ci.org/opentok/accelerator-textchat-js)
-[![GitHub release](https://img.shields.io/github/release/opentok/accelerator-textchat-js.svg)](./README.md)
-[![license MIT](https://img.shields.io/github/license/mashape/apistatus.svg)](./.github/LICENSE)
-[![npm](https://img.shields.io/npm/v/opentok-text-chat.svg)](https://www.npmjs.com/package/opentok-text-chat)
+The OpenTok Text Chat Accelerator Pack for JavaScript provides functionality you can add to your OpenTok applications that enables users to exchange text messages between mobile or browser-based devices.
 
------
+## Quick start
 
-# Accelerator TextChat for Javascript<br/>
+This section shows you how to use the accelerator pack.
 
-## Quick start
+### Prerequisites
 
-The OpenTok Text Chat Accelerator Pack provides functionality you can add to your OpenTok applications that enables users to exchange text messages between mobile or browser-based devices.
+The dependencies for this library are listed in the `package.json`:
+
+- jquery.
+- kuende-livestamp.
+- moment.
+- opentok-solutions-logging.
+- underscore.
 
-This section shows you how to use the accelerator pack.
 
-## Install
+### Install using a module bundler
+
+Start by installing the dependencies:
 
 ```bash
 $ npm install --save opentok-text-chat
 ```
 
-If using browserify or webpack:
+####Using a module bundler
```

Comment:
Thanks you're right, since this is a recent change in GitHub's Markup parsing engine some of the editing tools still parse this correctly (including mine), I'll search for one that does :+1: .